wagtail/admin/views/pages/delete.py

```diff
@@ -50,5 +77,10 @@ def delete(request, page_id):
             "page": page,
             "descendant_count": page.get_descendant_count(),
             "next": next_url,
+            "translation_count": translation_count,
+            "translation_descendant_count": sum(
+                [p.get_descendants().count() for p in pages_to_delete]
+            ),
+            "combined_subpages": page.get_descendant_count() + translation_count,
```

Comment:
This is extra. We only provide a message about translated pages, which doesn't include aliases